src/rules/date-rules.ts

```diff
@@ -3,6 +3,8 @@
 // and matching what date rules to use.
 export const DateRules = {
   REQUIRED: (v: string) => !!v || 'Select date',
+  RESOLUTION_REQUIRED: (v: string) => !!v || 'Resolution date is required',
+  SIGNATURE_REQUIRED: (v: string) => !!v || 'Signature date is required',
```

Comment:
I'm thinking these don't belong in DateRules...

Maybe under ResolutionRules and SignatureRules ?

Reply:
You are right, too specific! Will fix this

Reply:
Updated. Maybe future state we figure out how to pass arguments into the rules to make the return messages custom.